src/robotide/log/log.py

```diff
@@ -82,7 +82,8 @@ def _create_menu(self):
         self.register_action(ActionInfo(
             'Tools', 'View RIDE Log', self.OnViewLog, position=84))
 
-    def _log_message(self, log_event):
+    def _log_message(self, message):
+        log_event = message
```

Comment:
Kinda redundant. Why not just replace ``log_event`` with ``message`` everywhere in the method?

Reply:
I thought it could be a little confusing if replace all `log_event` to `message`.

But we should avoid redundant, updated.